content/docs/introduction/glossary.md

```diff
@@ -32,28 +32,33 @@ pull metrics from other systems and expose the metrics to Prometheus.
 ### Collector
 
 A collector is a part of an exporter that represents a set of metrics. It may be
-a single metric as part of direct instrumentation, or many metrics if it is pulling
-metrics from another system.
+a single metric if it is part of direct instrumentation, or many metrics if it is pulling metrics from another system.
 
 ### Direct instrumentation
 
-Direct instrumentation is when instrumentation is added inline as part the source code
+Direct instrumentation is instrumentation added inline as part the source code
 of a program.
 
 ### Exporter
 
 An exporter is a binary that exposes Prometheus metrics, commonly by converting
 metrics that are exposed in a non-Prometheus format into a format Prometheus supports.
 
+### Instance
+
+An individually scraped target, usually corresponding to a single process.
+
+### Job
+
+A collection of instances of the same type, for example replicated for scalability or reliability, is called a job.
```

Comment:
Not quite, as this is about targets rather than instances. "type" is also vague, the notion is more that it's the same binary and purpose.

Reply:
As above from the Concepts page.